sklearn/cluster/_spectral.py

```diff
@@ -21,6 +21,8 @@
 from ..neighbors import kneighbors_graph, NearestNeighbors
 from ..manifold import spectral_embedding
 from ._kmeans import k_means
+from ..utils._param_validation import Hidden, Interval, StrOptions, validate_params
+
```

Comment:
Remove this line because you should have only 2 lines before the function defintion.